### ISSUE TITLE
2.6.0-beta.2: IE11 loses space around equation

### ISSUE THREAD

**Issue by dpvc** [mathjax/MathJax#1338]
As reported at [math.stackexchange](http://meta.math.stackexchange.com/a/22207/7798), the expressions in IE11 are missing some space around them.

This is due to [this line](https://github.com/mathjax/MathJax/blob/2.6.0-beta.2/unpacked/jax/output/HTML-CSS/jax.js#L271).  It appears that there are two things involved:  (1) the `border-spacing:3px` isn't having any effect (so the `padding:-3px` is reducing the space uncompensated by the boarder), and (2) the body element has the focus initially, which triggers this CSS rule.

**Comment by dpvc** [comment]
I think the main solution is to remove the `border-size` and `padding` statements and go back to the tight bounding box.

But I still don't like the rule being triggered by the body element being focused.  The `*:focus .MathJax` rule was to handle the problem with links that contain math not having the right outline.  So perhaps a rule triggered by `*[outline] .MathJax` would do it.

**Comment by dpvc** [comment]
OK, `*[outline] .MathJax` doesn't work, but `body :focus .MathJax` seems to work better (this only triggers if a child of the body element is focused, not the body itself).

A similar change news to be made to CommonHTML as well.

**Comment by dpvc** [comment]
Also, `border-spacing` is the wrong attribute.  It should probably be `border-width`.  I'll have to check to see if that works, but it may be best to remove the extra spacing in any case.

**Comment by dpvc** [comment]
The `issue1338` branch of my fork of MathJax includes a fix for this.

**Comment by dpvc** [comment]
=> Merged.
